.github/workflows/pull-db-tests.yml

```
@@ -251,3 +263,16 @@ jobs:
         env:
           TAGS: bindata
           USE_REPO_TEST_DIR: 1
+
+  test-coverage-check:
+    if: needs.files-changed.outputs.backend == 'true' || needs.files-changed.outputs.frontend == 'true'
```

```suggestion
    if: needs.files-changed.outputs.backend == 'true'
```

Frontend coverage is a separate topic and not related.